stringzilla/pybind11.cpp

```diff
@@ -18,6 +18,7 @@ typedef SSIZE_T ssize_t;
 
 #include <utility> // `std::exchange`
 #include <limits>  // `std::numeric_limits`
+#include <cmath>
```

Comment:
What are you using from here?

Reply:
std::abs, it will be mentioned in the comment at the next commit.